static/app/components/organizations/pageFilterRow.tsx

```diff
@@ -12,7 +12,7 @@ type CheckboxRenderOptions = {
   checked?: boolean;
 };
 
-type Props = {
+type Props = React.HTMLAttributes<HTMLDivElement> & {
```

Comment:
Should we be using an interface instead of a union as HTMLAttributes is a chonky type?